dashboards-observability/public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -806,27 +807,73 @@ export const Explorer = ({
     );
   };
 
+  const testAntlr = (query: string) => {
+    const qm = new QueryManager();
+    // const pplQueryBuilder = new PPLQueryBuilder();
```

Comment:
Can remove this comment.

Reply:
these are old, outdated commits, which have been already removed in later commits.